common/AppRouter.tsx

```diff
@@ -39,14 +40,17 @@ export const AppRouter = () => {
                 <Switch>
                   {/* To avoid fiddling with layout we provide a complete route to home */}
                   <LayoutWithLocation>
-                    <Switch>
-                      <Route path={ROUTE_PATHS.ROOT.path} component={Home} exact={true} />
-                      <Route path={ROUTE_PATHS.HOME.path} component={Home} exact={true} />
-                      {APP_ROUTES.filter(route => !route.seperateLayout).map((config, idx) => (
-                        <PrivateRoute key={idx} {...config} />
-                      ))}
-                      <Route component={PageNotFound} />
-                    </Switch>
+                    {shouldShowError() && <Home />}
+                    {!shouldShowError() && (
```

Comment:
Lines 43 & 44 can be *terned* into a ternary ;)

Reply:
Or maybe, it may be a good idea to instead of just showing <Home /> under whatever the endpoint the user attempts to navigate to, maybe redirect to `/home`.

Reply:
Yeah, I'm beginning to think we should just redirect on error instead, that way we don't lock down the app.